app/scripts/services/stats.js

```diff
@@ -33,6 +33,7 @@ angular.module('stats', ['sql', 'health', 'tableinfo', 'nodeinfo'])
           setReachability(true);
         } else {
           data.version = null;
+          var storedURI = localStorage.getItem("crate.base_uri");
```

Comment:
yes, we should move the warning into the `baseURI` function
this should be the single place to obtain the URI from the storage